utils/sql/git/required/2023_01_18_bots_enable.sql

```diff
@@ -0,0 +1 @@
+UPDATE `rule_values` SET `rule_value` = 'true' WHERE `rule_name` = 'Bots:AllowBots';
```

Comment:
This shouldn't be in required because not everyone wants to enable bots.

The rule itself will be auto injected into the database at whatever default value as well